INSTALL.md

```diff
@@ -44,8 +44,8 @@ operating system. In particular, it does not include an automatic installer
 program.
 
 Installing the GAP distribution with all the packages and full data libraries
-takes about 1.6 GB of disk space and (except on Windows) will require both a C
-and a C++ compiler (gcc is recommended) to be installed on your system.
+and (except on Windows) will require both a C and a C++ compiler (gcc or clang
+are recommended) to be installed on your system.
```

Comment:
I feel pretty sure that this should still be 
```suggestion
is recommended) to be installed on your system.
```
Of course it might be my peculiar interpretation of the English language, but with an 'or' statement, I like to use the same verb conjugation that I would use if I 'expanded' the statement:
> gcc or clang is recommend → gcc is recommended or clang is recommended

I don't mind if you disagree and want to keep the sentence as is. Otherwise, no objections to this PR 🙂